### DISCUSSION TITLE
How to disable searching for new content

### DISCUSSION THREAD

**Discussion by ghost** [TeamNewPipe/NewPipe#10434] [Questions] [answered]
Say I would like to limit myself to watching already registered or imported channels, what is the key code to comment out to **disable the searching functionality** (searching for new videos or channels)?

**Comment by petlyh** [comment, marked as answer]
Probably the easiest way is to just remove the search button, since that's the only way of accessing the search functionality. You can do that by removing or commenting out the `item` element (lines 5 through 10) in [app/src/main/res/menu/menu_main_fragment.xml](https://github.com/TeamNewPipe/NewPipe/blob/dev/app/src/main/res/menu/menu_main_fragment.xml).

**Comment by ghost** [comment]
Thanks very much.
